frontend/public/components/persistent-volume-claim.jsx

```diff
@@ -17,21 +17,32 @@ const PVCStatus = ({pvc}) => {
 };
 
 const Header = props => <ListHeader>
-  <ColHead {...props} className="col-sm-4 col-xs-6" sortField="metadata.name">Name</ColHead>
-  <ColHead {...props} className="col-sm-4 col-xs-6" sortField="metadata.namespace">Namespace</ColHead>
-  <ColHead {...props} className="col-sm-4 hidden-xs" sortField="status.phase">Status</ColHead>
+  <ColHead {...props} className="col-sm-2 col-xs-6" sortField="metadata.name">Name</ColHead>
+  <ColHead {...props} className="col-sm-2 col-xs-6" sortField="metadata.namespace">Namespace</ColHead>
+  <ColHead {...props} className="col-sm-2 hidden-xs" sortField="status.phase">Status</ColHead>
+  <ColHead {...props} className="col-sm-3 hidden-xs" sortField="spec.volumeName">Persistent Volume</ColHead>
+  <ColHead {...props} className="col-sm-3 hidden-xs" sortField="spec.resources.requests.storage">Requested</ColHead>
 </ListHeader>;
 
 const kind = 'PersistentVolumeClaim';
 const Row = ({obj}) => <div className="row co-resource-list__item">
-  <div className="col-sm-4 col-xs-6">
+  <div className="col-sm-2 col-xs-6">
     <ResourceLink kind={kind} name={obj.metadata.name} namespace={obj.metadata.namespace} title={obj.metadata.name} />
   </div>
-  <div className="col-sm-4 col-xs-4 co-break-word">
+  <div className="col-sm-2 col-xs-6 co-break-word">
     <ResourceLink kind="Namespace" name={obj.metadata.namespace} title={obj.metadata.namespace} />
   </div>
-  <div className="col-sm-4 hidden-xs">
-    <PVCStatus pvc={obj} />
+  <div className="col-sm-2 hidden-xs">
+      {<PVCStatus pvc={obj} />}
+  </div>
+  <div className="col-sm-3 hidden-xs">
+    { !_.isEmpty(obj.spec.volumeName) ?
+      <ResourceLink kind="PersistentVolume" name={obj.spec.volumeName} title={obj.spec.volumeName} />          :
+      <div className="text-muted" key="0">No Persistent Volume</div>
+    }
+  </div>
+  <div className="col-sm-3 hidden-xs">
+    {obj.spec.resources.requests.storage}
```

Comment:
Will this definitely always be set? If not, it could be a runtime error. Maybe `_.get` here?

```suggestion
    {_.get(obj, 'spec.resources.requests.storage', '-')}
```

Reply:
i don't think it will always come, i will use _get